docs/docs/admin/installation.mdx

```diff
@@ -71,6 +72,40 @@ Anubis uses these environment variables for configuration:
 
 For more detailed information on configuring Open Graph tags, please refer to the [Open Graph Configuration](./configuration/open-graph.mdx) page.
 
+### Using Base Prefix
+
+The `BASE_PREFIX` environment variable allows you to run Anubis behind a path prefix. This is useful when:
+
+- You want to host multiple services on the same domain
+- You're using a reverse proxy that routes based on path prefixes
+- You need to integrate Anubis with an existing application structure
+
+For example, if you set `BASE_PREFIX=/myapp`, Anubis will:
+
+- Serve its challenge page at `/myapp/` instead of `/`
+- Serve its API endpoints at `/myapp/.within.website/x/cmd/anubis/api/` instead of `/.within.website/x/cmd/anubis/api/`
+- Serve its static assets at `/myapp/.within.website/x/cmd/anubis/` instead of `/.within.website/x/cmd/anubis/`
+
+When using this feature with a reverse proxy:
+
+1. Configure your reverse proxy to route requests for the specified path prefix to Anubis
+2. Set the `BASE_PREFIX` environment variable to match the path prefix in your reverse proxy configuration
+3. Ensure that your reverse proxy preserves the path when forwarding requests to Anubis
+
+Example with Nginx:
+```nginx
+location /myapp/ {
+    proxy_pass http://anubis:8923;
```

Comment:
In apache this would be:

```
ProxyPass /myapp http://127.0.0.1:3334/myapp
ProxyPassReverse /myapp http://127.0.0.1:3334/myapp
```

So i think in nginx you would also need the /myapp

```proxy_pass http://anubis:8923/myapp;```

Reply:
Oh interesting. Yeah, you're right on that part if my assumptions are right. 

I'll need some real world testing though

Reply:
Sure though this is how I have mine setup.